waspc/examples/todoApp/todoApp.wasp

```diff
@@ -10,13 +10,17 @@ app todoApp {
   ],
```

Comment:
This is a good question, I wonder if we should ask the team in Discord since it involves a process update. But in general, I do think if we know it will be a major bump, that we can update the changelog, cabal, and all apps at once. I think that should be fine?

Reply:
So you mean updating all of that once time for release comes? Or as we do the commit that introduces a change?